ios/RNSScreenWindowTraits.mm

```diff
@@ -4,6 +4,25 @@
 
 @implementation RNSScreenWindowTraits
 
+- (instancetype)init
+{
+#if !TARGET_OS_TV
+  dispatch_async(dispatch_get_main_queue(), ^{
+    [[UIDevice currentDevice] beginGeneratingDeviceOrientationNotifications];
+  });
+#endif
+  return self;
+}
+
+- (void)dealloc
+{
+#if !TARGET_OS_TV
+  dispatch_sync(dispatch_get_main_queue(), ^{
+    [[UIDevice currentDevice] endGeneratingDeviceOrientationNotifications];
+  });
+#endif
+}
```

Comment:
Haven't tested it yet, but as we talked - I believe we do not instantiate this class -> this code is dead. 

I'm leaving this note so we don't forget and merge it by accident.